### DISCUSSION TITLE
Extra Thinking Cap: How can local communities take ownership of technology?

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#122] [Extra Credit]
# Due 4/27 before class

Reflect on the guest lecturers and how we are thinking about collecting survey data. How do we avoid extracting information from communities and walking away after the course is over? What ideas or issues do you have with simply giving an application to an organization or local people?

### Guidelines
- Respond to the prompt with at least 5 sentences.
- Not required to respond to other students.
- This assignment is worth 15 points (not 30).

## Submission: 
Post as a comment below.

**Comment by okcirE** [comment]
I believe that one of the ways to preserve the integrity and usefulness of information that has been extracted from local bodies is by meaningful implementations and disseminating insights obtained from the data. We need to constantly remind ourselves that people are the focus and that data should work for the people's good and not the other way around. Education is also important in the case of teaching insights gained from the collected data to better the targeted community's understanding of their current state and ways to improve. A big issue that was raised from today's lecture was sustainability and I definitely agree with this because simply handing ownership of an application to an org/local body doesn't mean that the app will remain to live and serve the people. Local bodies/orgs might not have the manpower and/or capital to maintain the efficacy of the intended product. Therefore, a solution to this problem can be a collaboration between NPOs that focus on creating impact in their communities and NPOs that are focused on creating software for the people. In this way, I believe that sustainability can be achieved.

**Comment by rachan2023** [comment]
To avoid extracting information from communities and walking away after the course is over, we need to continue to think about the community’s needs and how this application can continue to benefit them in the long run. It is especially important to take a user-centered approach to understand the issues that local community members are facing as well as finding a sustainable way to attain a solution. When taking a user-centered approach, we should talk to local community members or think from their perspective while also being conscious of the biases that we hold. It is especially important that when we build the application, we are continuously asking ourselves, “is this what the community needs?,” “will this hurt or harm them?,” and “how will this application continue to benefit them in the future?.” After this application is created, we can’t simply give the application to the organization and or local people and leave because they may not have the technical skills to maintain the website or update the application if there are bugs or they may not know what to do with the application itself. In lecture, Nina Kin mentioned that Code for America, a non-profit, created an online application for people to apply for CalFresh, which bridges the gap between county officials issuing food stamps and potential CalFresh recipients. This exemplifies the importance of working with organizations that understand the needs of their community and creating an application that will benefit them in the long-run.

**Comment by annyrviloria** [comment]
One situation I often encountered at my last job was that college students who needed to fulfill some community engagement requirement or project would approach the NGO I worked for in hopes to support our work. Because we were a non-profit (ahem underfunded ahem), we would take any help we could -- but over time we came to the conclusion that despite all the best intentions many volunteers had, our communities were being effectively treated as testing grounds for a bunch of pet projects that had no real future. I think this relates to the question of how to avoid simply extracting information with people and then walking away. Having a sustainable plan from the beginning of the project is the only way to ensure that we are not replicating the same power dynamics we seek to challenge via our applications. Moreover, we need to set very clear and realistic expectations with the community we are partnering with regarding the true trajectory of our application. It would be disrespectful and unethical to obtain permission to collect data from a given community in exchange of something we cannot realistically do. Thus, mutual respect and emphasis on sustainability are the central principles upon which we should seek to build trust with the community we want to serve through our project. More broadly, I think we should also embrace humility as a value to bring into the work we are embarking on. Admitting we don't have all the answers and relying on community experts to learn about their own lived experience is a simple yet effective way to elevate local knowledge and leadership above the desires we as "project leads" might bring to the table.

**Comment by Wny-Duong** [comment]
- How do we avoid extracting information from communities and walking away after the course is over? 

Honestly, it feels pretty inevitable that the sort of extraction of information and walking away from communities is going to occur again and again despite whatever safeguards or methods you teach in order to prevent it. We can say things about keeping the community in mind and treating them as human beings instead of just sources of information, but circumstances do and will occur where a project may fall apart due to a lack of interest from the original developer.  It's a very unfortunate thing, but it happens, so the best thing to do is to mitigate damage done from extraction and walking away.

It'd be best to ensure that the collection of information is at least impactful. At the very least, it's important to let members of the community know how the data was planned to be used, what else can be done with it, and provide them with the starting points to continue using that information effectively (perhaps follow-up contacts with people who've showed interest in using the information to continue development). Of course, even that kind of thing is a pretty demanding and out-of-touch opinion to hold since it's very likely that an in-need community is not going to have the free time, resources, and ability to develop a technical background in the same way an "expert developer" or "field researcher" would since those people's employment are based on those tasks. 

 In short, the community should be "convinced" that the effort to extract information from them was done with the best of intentions (they shouldn't feel exploited, more importantly) and has the capability of doing good (the community shouldn't feel like data collection was a waste of their time), even if the original extractor of data was unable to do anything with it.

- What ideas or issues do you have with simply giving an application to an organization or local people?

The most effective applications are typically more specific in their scope and get developed over time as additional support is gathered to scale them to a larger size as seen with GetCalFresh. If that's the case, I'd imagine that you'd have to do two things.

The first is to give the application to people who're actively interested in developing it and have an understanding of the disadvantaged communities those applications are trying to support. In other words, giving them to community experts with the technical skills and understanding of how to apply them to suit community needs. This, however, might lead to an issue where you, as the authority handing off the project, are reinforcing structures of power you might not properly understand and can do some harm as you're treating yourself as an authority in identifying who best supports your agendas in the communities you should be trying to support instead of guide.

To remedy this issue, it'd be ideal to foster the interest in technology and issue area of civil society being targeted by the technology directly with the typical members of the in-need community who have some interest. Giving them the tools, resources, contacts, methodologies, and enthusiasm for development of civil society in order to find and resolve the problems plaguing their community is going to be more sustainable in the long run than just leaving behind a tool. In relation to an issue mentioned prior about a community that does not have the leisure time to pursue or gamble on web development or the use of new high-tech tools, it's equally important to reduce the barriers for acquiring the interest in technology for civic good, although I'm personally not sure what that'd look like. 

Basically, you want to leave behind a network/community of people with the ability and want to identify and resolve problems, one that's capable of growing on its own. That's a lot better than just dropping some app and saying that a community's problem is solved and they can move on.

**Comment by nnhimichelle** [comment]
The biggest thing to do to avoid extracting information from communities and walking away after the course is over is to have a  long-term plan that acquires simple and realistic approaches that are effective at the same time. One way is to make sure that any use of data is for the purposes of education, and even further to make that education not only transparent to all users but also accessible across all open sources so that it can reach other communities that may need it. With the targeted communities, they can also insert their own input so that they can educate each other and make their insight of information useful for one another. Simply giving an application to an organization or local people will put progression at a stopping point if people do not have the skills or even the technology itself to keep up with the application in terms of updates, troubleshooting, and more. It’s similar to extracting information from communities and walking away afterwards. There should be a long-term effort of working together between communities to help each other learn and work through the application and the malfunctions that may occur with the data and software in the application. In such case, this emphasizes the evaluation of community or targeted audience needs, so that they can benefit from any application throughout the long-term.

**Comment by ninalong** [comment]
I think that the answer to developing sustainable tools that remain benefiting the community is inherent in the question, giving local communities ownership of technology. Often, well-meaning folks seek to help communities through a specific initiative, but despite their best intentions, their projects are doomed to fail from the beginning. If projects are not embedded into long-lasting frameworks that will maintain tech, continue outreach, and adjust tools to preserve their value in a changing landscape, they will always fade into obscurity and "abandon" the groups they were meant to serve. Local ownership, potentially in the form of trusted community based organizations, entails a focused commitment on the community and a constant presence. Of course, local institutions can experience issues with longevity as well (budget cuts, grant funding ending, etc.) but they can be some of the best options. My group's project is embedded within the WCK Neighborhood Council, ensuring that the project will go on beyond our involvement.

**Comment by brillianz** [comment]
Over the course of my education, I have come to learn the importance of manipulating the institution so that it works towards our goals and worldviews. Academic institutions, much like research methods, are exploitative in nature and look to extract information as quickly as they can. Instead of giving up on these systems completely; however, I still understand them in terms of their potential towards systemic change. Accordingly, this should expand the access of leadership roles within tools such as survey methods so that the communities' desires are prioritized in the findings. Indigenous scholar Eve Tuck identifies this research approach as a "desire-based" approach, which guides the research according to what the community sees as most important to their lives. 

The continued relationship with these communities is largely viewed as "unprofessional" in research, but community-based researchers suggest otherwise. Whether providing knowledge shares or introducing workshops for tangible resources in the future, the lens in research has to extend itself to acknowledging the importance of giving back for the data that's provided by the community.  To sum, collaboration with communities must be made before, during, and following the data collection process. It is with this approach that the most authentic, humanizing research design can be achieved.
